js/index.js

```diff
@@ -38,5 +38,128 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+//nav stuff
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let navItem1 = document.querySelector("nav a:first-child");
+navItem1.textContent = siteContent["nav"]["nav-item-1"];
+
+let navItem2 = document.querySelector("nav a:nth-child(2)");
+navItem2.textContent = siteContent["nav"]["nav-item-2"];
+
+let navItem3 = document.querySelector("nav a:nth-child(3)");
+navItem3.textContent = siteContent["nav"]["nav-item-3"];
+
+let navItem4 = document.querySelector("nav a:nth-child(4)");
+navItem4.textContent = siteContent["nav"]["nav-item-4"];
+
+let navItem5 = document.querySelector("nav a:nth-child(5)");
+navItem5.textContent = siteContent["nav"]["nav-item-5"];
+
+let navItem6 = document.querySelector("nav a:nth-child(6)");
+navItem6.textContent = siteContent["nav"]["nav-item-6"];
+
```

Comment:
Look into how you could have selected all of the nav items at once using `querySelctorAll()` then accessed each individual nav item with bracket notation.  That would allow you to use methods like `forEach()` to iterate through each nav item at once.